client/components/product-icon/test/test.jsx

```diff
@@ -0,0 +1,104 @@
+jest.mock( 'lib/abtest', () => ( {
```

Comment:
Why is that necessary?

Reply:
I copied the whole test file from `PlanIcon` and haven't checked it. Will double check 👍